src/components/Subscription/CalendarOfInterestSubscriptionPicker.vue

```diff
@@ -92,21 +97,19 @@ export default {
 		this.loading = false
 	},
 	methods: {
-	  async subscribe(calendar) {
+		async subscribe(calendar) {
 			try {
-			  this.subscribing[calendar.source] = true
+				this.subscribing[calendar.source] = true
 
 				await this.$store.dispatch('appendSubscription', {
-					displayName: t('calendar', 'Holidays in {region}', {
-					  region: calendar.country,
-					}),
+					displayName: t('calendar', calendar.name),
```

Comment:
Since the calendar name is variable, this will not need a translation unless you add a string such as "Calendar of interest {calendar.name}" or some such

Reply:
I'm not quiet sure but it seems to me you could create a translation like `"Public holidays in %s": "Feiertage in %s"` to translate every name with that prefix.